cdisc-org/cdisc-rules-engine
# Med-RT validation support

Steps to test:

* Unit tests pass
* Run previous dictionary test rules with the `--med-rt` argument pointing to a directory containing a valid medrt DTS xml file. (see attached) and verify the rule runs correctly.
